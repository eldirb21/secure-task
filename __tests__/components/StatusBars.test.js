import React from 'react';
import renderer from 'react-test-renderer';

import StatusBars from '@components/statusBars';

describe('StatusBars component', () => {
  it('renders StatusBars component correctly', () => {
    const backgroundColor = 'blue';
    const tree = renderer
      .create(
        <StatusBars backgroundColor={backgroundColor} translucent={true} />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
