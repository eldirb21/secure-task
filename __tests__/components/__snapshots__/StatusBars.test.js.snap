// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBars component renders StatusBars component correctly 1`] = `
<View
  style={
    [
      {
        "height": null,
      },
      {
        "backgroundColor": "blue",
      },
    ]
  }
>
  <RCTSafeAreaView />
</View>
`;
